app/helpers/reports_helper.rb

```diff
@@ -108,4 +108,56 @@ def self.fake_genus(tax_id, tax_info)
     tax_info[:genus_tax_id] = fake_genus_id
     fake_genus_info
   end
+
+  def report_csv_from_params_v2(sample, params)
+    background_id = params[:background_id] || sample.default_background_id
```

Comment:
Please use the same parameters and the same code as report_v2 for selecting the background and deciding to output something.
Since we will move the CSV formatting below to the pipeline report service, we also do not need an helper for this.